README.md

```diff
@@ -1,8 +1,64 @@
-# Test Repo
+# EnChord
 
-The chopsticks are the best female developer duo.
+**EnChord** is a perfect tool to develop your music dictation skills. Challenge yourself with different levels of quizzes and follow your progress by checking your highest score and average test results. You can also review what are the frequently mistaken interval types.
```

Comment:
I'm probably misunderstanding it, but I think dictation refers to writing down what you hear - would "aural skills" be a better fit here?

Reply:
That is actually true! Musical dictation mostly includes marking down what you hear. I will update it now. Also, I was thinking to include some of the issues you raised in #40 in this PR,  except changing the deployed website name. What do you think? I thought, these are too minors (doesn't mean they are not important) to make another PR 😄

Reply:
(You gotta sell what you made regardless of the quality right 🤣 🤣 . So it is a perfect tool for me. Or at least will be. haha)

Reply:
Thanks for updating the project name and checking off the relevant boxes ❤️